### ISSUE TITLE
`sep` parameter is inconsistent with builtin `print`

### ISSUE THREAD

**Issue by darrenburns** [willmcgugan/rich#13]
`sep` behaves differently from builtin print. I don't know if this is intentional or not, but the behaviour is slightly different from how the docs describe it. (I didn't expect the `sep` to be added to the *end* of the string, only *between* the printed items).

**Example:**
```python
print(".", ".", sep="a")
console.print(".", ".", sep="a")
```

**Output:**
```
.a.
.a.a
```

**Expected output:**
```
.a.
.a.
```

**Comment by willmcgugan** [comment]
Yeah, should be the same as print.

**Comment by willmcgugan** [comment]
Should be fixed in v0.6.0.

How is integration with Ward coming on?
